### PR TITLE
Add missing animation speeds for models

### PR DESCRIPTION
**Pull request by Miepee**
These were supposed to be updated with #5841

### CODE REVIEW

**Review by dyceron — APPROVED** [review]

File: randovania/games/am2r/pickup_database/model_data.json [hunk 1]
```diff
@@ -66,6 +67,7 @@
     "sItemAmberEchoes": 0.1666,
     "sItemAnnihilatorEchoes": 0.2,
     "sItemBeamAmmoEchoes": 0.2,
+    "sItemCannonBall": 0.1666,
```
Comment (dyceron) [inline randovania/games/am2r/pickup_database/model_data.json:70]: Missing Echoes

**Review by dyceron — CHANGES_REQUESTED** [review]
Oh I found another missed line. Dark Beam is missing Echoes

**Comment by dyceron** [comment]
Also is Arftifact supposed to say Prime?

**Comment by Miepee** [comment]
The sprites are called like that in YAMS:
grafik
grafik

**Comment by dyceron** [comment]
> The sprites are called like that in YAMS: grafik grafik

Ah OK. Inconsistent with the rest but if that's what it's called then it's finr.

**Pull request merged**